web/server/codechecker_server/api/report_server.py

```diff
@@ -1962,45 +1962,29 @@ def getRunResults(self, run_ids, limit, offset, sort_types,
                     ReportAnnotations.value)])).label(f"annotation_{col}")
 
             if report_filter.isUnique:
+                # A report annotation filter cannot set in WHERE clause if we
+                # use annotation parameters in aggregate functions to create
+                # pivot table. Instead filtering report annotations in WHERE
+                # clause, we should use HAVING clause only for filtering
+                # aggregate functions.
+                # TODO: Fixing report annotation filter in every report server
+                # enpoint function.
+                report_filter_annotations = report_filter.annotations
```

Comment:
```suggestion
                annotations_backup = report_filter.annotations
```
Maybe such a name could cause less misunderstanding compared to that one-character difference.